tests/components/met/test_config_flow.py

```diff
@@ -71,6 +72,26 @@ async def test_create_entry(hass: HomeAssistant) -> None:
     assert result["data"] == test_data
 
 
+async def test_create_entry_us_customary_system(hass: HomeAssistant) -> None:
```

Comment:
I'm not sure this test adds much value, it's not testing what's changed in the PR?

Reply:
Removed this test

Reply:
We prefer that commits are not squashed while working on a PR, we squash on merge instead 👍